packages/client/src/Conversation.tsx

```diff
@@ -235,24 +265,25 @@ function Presentable({
           </div>
         }
       />
-      <CardContent>
-        {presentable.contents.map((content, i) => {
-          if (editing) {
-            return (
-              <EditForm
-                accountId={accountId}
-                conversationId={conversationId}
-                contentToEdit={content}
-                onComplete={() => {
-                  setEditing(false)
-                }}
-              />
-            )
-          } else {
+      <Collapse in={expanded} timeout="auto" unmountOnExit>
```

Comment:
`unmountOnExit` removes child elements from the DOM after the transition ends, as opposed to hiding them. One implication is that when the element is shown again its state is reset, and any queries it runs on mount need to re-run. In the case of presentables if you start editing a presentable, change the text, collapse, and then expand your changes will be lost. So I think we would be better of removing `unmountOnExit`.